layout-server/src/test/java/org/finos/vuu/layoutserver/integration/LayoutIntegrationTest.java

```diff
@@ -32,14 +39,16 @@
 @ActiveProfiles("test")
 public class LayoutIntegrationTest {
 
-    private static final String DEFAULT_LAYOUT_DEFINITION = "Default layout definition";
+    private static final String DEFAULT_LAYOUT_DEFINITION_STRING = "{\"id\":\"main-tabs\"}";
+    private static final String DEFAULT_UPDATED_LAYOUT_DEFINITION_STRING = "{\"id\":\"updated-main-tabs\"}";
```

Comment:
Minor semantic thing that's not going to stop me from approving: I would remove the word "default" and just call this `UPDATED_LAYOUT_DEFINITION_STRING`.